bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/AudioFormat.java

```diff
@@ -24,19 +24,23 @@
 public class AudioFormat {
 
     // generic mp3 format without any further constraints
-    public static AudioFormat MP3 = new AudioFormat(AudioFormat.CONTAINER_NONE, AudioFormat.CODEC_MP3, null, null, null,
+    public static final AudioFormat MP3 = new AudioFormat(AudioFormat.CONTAINER_NONE, AudioFormat.CODEC_MP3, null, null,
+            null,
```

Comment:
please run the auto formatter for this file once. Lines 28 & 29 should be on one line. And so should the other "hanging" parameters.

Reply:
Done :)